packages/venia-concept/src/components/CreateAccount/createAccount.js

```diff
@@ -74,7 +74,10 @@ class CreateAccount extends Component {
 
     render() {
         const { errorMessage, handleSubmit, initialValues, props } = this;
-        const { classes } = props;
+        const { classes, isSignedIn } = props;
+        if (isSignedIn) {
+            return <Redirect to="/" />;
```

Comment:
This throws a `ReferenceError: Redirect is not defined` error.

Be sure to include `Redirect` in the `import` statements at the top of the file 👍